06.bowling_object/class/game.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+require_relative './last_frame'
+require_relative './shot'
+
+class Game
+  def initialize
+    @frames = []
+  end
+
+  def play(marks)
+    shots = Shot.shot_factory(marks)
+
+    @frames = []
+    (1..10).each do |index|
+      frame = index != 10 ? Frame.new : LastFrame.new
+      (1..3).each do |_|
```

Comment:
```suggestion
      (1..3).each do
```

使わないならパラメータは省略可能です。

あと、こういう書き方もあります。

```suggestion
      3.times do
```